### DISCUSSION TITLE
Overclock possible?

### DISCUSSION THREAD

**Discussion by Bahazzie** [Joshua-Riek/ubuntu-rockchip#602] [General]
Right now my OPI5 sits fanless in an alu case. Works brilliant, I hate moving parts and noise, and this doesn't throttle a bit even stresstesting for an hour. Right now I disabled Wayland, because I want to have RealVNC working. Latest Geekbench6 speed test was 800/3050. Not bad and a bit better than the latest Armbian I tried. But according to some folks the 2.3Mhz limit can be removed, resulting in 2.4 Mhz max. Is this possible in JRUbuntu? 

Second question: what I don't quite understand is that some software like Mu (Python editor) runs fine on rpi, but here the Gnome software center says not compatible. Arm64 is arm64, no?

Bas

**Comment by HZFrodo** [comment]
See https://github.com/armbian/build/pull/4733
